pkgs/tools/misc/didyoumean/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, xorg
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "didyoumean";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "hisbaan";
+    repo = "didyoumean";
+    rev = "v${version}";
+    sha256 = "sha256-t2bmvz05vWIxQhC474q/9uky1kAQoFN8Z+qflw5Vj68=";
+  };
+
+  cargoSha256 = "sha256-4DbziI9enib4pm9/P4WEu15glIxtejaV2GCqbzuxxyw=";
+
+  buildInputs = with xorg; [ libxcb ];
+
+  meta = with lib; {
+    description = "A CLI spelling corrector for when you're unsure. Written in Rust";
```

Just one sentence, no trailing dot, written in rust does not matter for the user
```suggestion
    description = "A CLI spelling corrector for when you're unsure";
```